services/gitea/tasks/create_user.yml

```diff
@@ -0,0 +1,64 @@
+- name: "Create Gitea user(s)"
+  uri:
+    url: '{{ gitea_api_url }}/admin/users'
+    method: POST
+    body_format: json
+    status_code: 201
+    force_basic_auth: yes
+    url_username: '{{ gitea_admin_user }}'
+    url_password: '{{ gitea_admin_passwd }}'
+    body:
+      username: '{{ user.name }}'
+      password: '{{ user.password }}'
+      email: '{{ user.email }}'
+      admin: '{{ "true" if user.admin else false }}'
+      must_change_password: false
+    return_content: true
+  until: _response.status == 201
```

Comment:
this should be changed to something more broad. e.g. if you'd be getting a 4xx that's a client error, and not worth retrying

maybe `until: _response.status != -1 or _response.status >= 500` ?

Reply:
Could add this for other requests as well